internal/errors/errors.go

```diff
@@ -0,0 +1,48 @@
+package errors
+
+import "errors"
+
+var (
+	// ErrTokenNotFound creates a new NotFound error.
+	ErrTokenNotFound = errors.New("token not found")
+	// ErrTokenExpired creates a new ErrTokenExpired error.
+	ErrTokenExpired = errors.New("token expired")
+	// ErrTokenInactiveYet creates a new ErrTokenInactiveYet error.
+	ErrTokenInactiveYet = errors.New("token not active yet")
+	// ErrTokenSignatureInvalid creates a new ErrTokenSignatureInvalid error.
+	ErrTokenSignatureInvalid = errors.New("signature invalid")
+	// ErrTokenContainsInvalidSegments creates a new ErrTokenContainsInvalidSegments error.
+	ErrTokenContainsInvalidSegments = errors.New("token contains an invalid number of segments")
+	// ErrorTokenMalformed creates a new ErrorTokenMalformed error.
+	ErrorTokenMalformed = errors.New("token malformed")
+)
+
+// IsTokenNotFound returns true if the specified error is an ErrTokenNotFound.
+func IsTokenNotFound(err error) bool {
+	return err == ErrTokenNotFound
+}
+
+// IsTokenExpired returns true if the specified error is an ErrTokenExpired.
+func IsTokenExpired(err error) bool {
+	return err == ErrTokenExpired
+}
+
+// IsTokenInactiveYet returns true if the specified error is an ErrTokenInactiveYet.
+func IsTokenInactiveYet(err error) bool {
+	return err == ErrTokenInactiveYet
+}
+
+// IsErrTokenSignatureInvalid returns true if the specified error is an ErrTokenSignatureInvalid.
+func IsErrTokenSignatureInvalid(err error) bool {
+	return err == ErrTokenSignatureInvalid
+}
+
+// IsErrTokenContainsInvalidSegments returns true if the specified error is an ErrTokenContainsInvalidSegments.
+func IsErrTokenContainsInvalidSegments(err error) bool {
+	return err == ErrTokenContainsInvalidSegments
+}
+
+// IsErrorTokenMalformed returns true if the specified error is an ErrorTokenMalformed.
+func IsErrorTokenMalformed(err error) bool {
+	return err == ErrorTokenMalformed
+}
```

Comment:
please be consistent, example:
- use the complete name of Error, instead of Err
- use it in every method, or don't use it at all. since this is in the errors package, i would say you can drop it at all
- if using Error in the name of the method, i prefer IsSomethingError, so Error in the end instead of the beginning

Reply:
I agree.
thanks Portela! ❤️